ROADMAP.md

```diff
@@ -10,18 +10,16 @@ The version of the spec that rkt implements can be seen in the output of `rkt ve
 **Note that until communicated otherwise, the rkt CLI and API are considered unstable and subject to change.**
 Backwards-compatibility is a priority for the v1.0.0 release.
 
-### rkt v0.14 (December)
-- different shared namespace execution modes [#1433](https://github.com/coreos/rkt/issues/1433)
-  - `rkt fly`, suitable for running Kubelet
```

Comment:
these are not solved, and there's no `rkt fly` command yet.
I'd put `rkt fly` to v0.15.0 and the *different shared namespace execution modes* even v1.0, as that is a rather big project.